.trivyignore.yaml

```diff
@@ -1,9 +1,22 @@
+# See: https://aquasecurity.github.io/trivy/v0.56/docs/configuration/filtering/#trivyignore
+# for details on formatting this file.
+#
 # List of vulnerabilities to ignore for the trivy scan
+
 # Please add safelists in the following format to make it easier when checking
 # Package/module name: URL to vulnerability for checking updates
 #  Versions:     URL to the version history
 #  Dependencies: Name of any other packages or modules that are dependent on this version
 #                 Link to the dependencies for ease of checking for updates
 #  Issue:         Why there is a finding and why this is here or not been removed
 #  Last checked:  Date last checked in scans
-#The-CVE-or-vuln-id # Remove comment at start of line
+
+# vulnerabilities:
+#   - id: CVE-1234-5678
+
+secrets:
+  # PyJWT has an example with a fake JWT that Trivy flags.
+  # see: https://github.com/aquasecurity/trivy/discussions/5772
+  id: jwt-token
+  paths:
+    - "/api/.venv/lib/python3.13/site-packages/PyJWT-2.9.0.dist-info/METADATA"
```

Comment:
very rude that trivy won't accept a wildcard in here

Reply:
Oof... so this is going to break every time we upgrade Python or PyJWT?

Reply:
yep!

Reply:
While their docs don't mention wildcards, they might work? https://github.com/aquasecurity/trivy/issues/5120

Will test

Reply:
Unfortunately, wild cards do not work. So yeah, any version upgrade will break this.

Skipping directories in the scan technically could work with wildcards: https://aquasecurity.github.io/trivy/v0.56/docs/configuration/skipping/ - as long as we're fine with ignoring everything for this singular metadata file.

I'll look into it more tomorrow

Reply:
I think setting up skips like that requires a configuration file: https://aquasecurity.github.io/trivy/v0.56/docs/references/configuration/config-file/

We currently set some of these values with env vars. I think the right way would be to setup a config file to use, but that is definitely veering outside of the scope of this ticket, so I'll just revert it back to the direct path.